skrub/_datetime_encoder.py

```diff
@@ -45,35 +77,44 @@ class DatetimeEncoder(BaseEstimator, TransformerMixin):
     extract_until : {"year", "month", "day", "hour", "minute", "second",
         "microsecond", "nanosecond", None}, default="hour"
         Extract up to this granularity.
```

Comment:
if it is a "granularity" maybe "resolution" or "granularity" could be a better parameter name? "until" sounds more like we stop at a certain date